lib/report_email_sender.rb

```diff
@@ -0,0 +1,41 @@
+class ReportEmailSender
```

Comment:
not using [ActionMailer](http://guides.rubyonrails.org/action_mailer_basics.html)?